src/Platform.Handlers/src/Xamarin.Platform.Handlers/Hosting/HandlerServiceProvider.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Xamarin.Platform.Hosting
+{
+	class HandlerServiceProvider : IHandlerServiceProvider
```

Comment:
This code seems to be based on an older version of our Registrar that we had inside the handlers. 

Should we update this code? or just have this code call into the Registrar? 

The factory code added as part of the renderer shim code needs to be accounted for yes?